great_expectations/cli/cli_logging.py

```diff
@@ -4,7 +4,7 @@
 warnings.filterwarnings("ignore")
 
 logging.getLogger(
-    "great_expectations.datasource.generator.in_memory_generator"
+    "great_expectations.datasource.batch_kwarg_generator.in_memory_batch_kwarg_generator"
```

Comment:
In Memory Generator has already been removed, the CLI shouldn't try to touch its (nonexistent) logger

Reply:
Fixed